components/python/xattr/Makefile

```diff
@@ -0,0 +1,44 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2020 Nona Hansel
+#
+
+# Component uses ABI3 naming.
+PYTHON3_SOABI=abi3
+
+BUILD_BITS=		64
+BUILD_STYLE=		setup.py
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		xattr
+COMPONENT_VERSION=	0.9.7
+COMPONENT_SUMMARY=	'Extended attributes library wrapper for Python'
+COMPONENT_PROJECT_URL=	https://pyxattr.k1024.org/
```

Comment:
This page holds outdated information. https://github.com/xattr/xattr seems to be a more viable one.